includes/Services/StatusService.php

```diff
@@ -181,11 +186,28 @@ public static function track(): void {
 		}
 
 		// Ignore if the request is not for the onboarding page.
-		if ( isset( $_GET['page'] ) && WP_Admin::$slug === \sanitize_text_field( $_GET['page'] ) ) {
+		if ( isset( $_GET['page'] ) && \sanitize_text_field( $_GET['page'] ) === WP_Admin::$slug ) {
 			return;
 		}
 
 		// Handle abandoned event.
 		self::handle_abandoned();
 	}
+
+	/**
+	 * Save onboarding site information to database option for other modules to access.
+	 *
+	 * @return void
+	 */
+	public static function save_site_info(): void {
+		$site_info = array();
+
+		// Get experience level and site type from ReduxStateService
+		$data = ReduxStateService::get( 'input' );
+		$site_info['experience_level'] = $data['experienceLevel'] ?? 3;
```

Comment:
I see that we're falling back on `int` 3 but the `experienceLevel` is stored as a string, see how it's being translated here: https://github.com/newfold-labs/wp-module-onboarding-data/blob/main/includes/Services/FlowService.php#L442-L453

However, that being said, i'd rather store it as string since that's more readable, so you'll just have fallback to 'advanced' instead of number 3.